README.md

```diff
@@ -33,7 +33,8 @@ terminus(server, {
 
   // cleanup options
   timeout: 1000,                   // [optional = 5000] number of milliseconds before forcefull exiting
-  onSigterm,                       // [optional] cleanup function, returning a promise
+  killSignal,                      // [optional = 'SIGTERM'] what signal to listen for relative to shutdown
```

Comment:
naming nit: `kill` is a little confusing because there's a `SIGKILL` and userspace applications are unable to catch or handle that signal.  How about simply `signal` and `onSignal`?

Reply:
sounds great! @vernak2539 what do you think?

Reply:
so it was `signal` at one point, but was asked to be changed to`killSignal`. I do understand your point, but in this case isn't the signal being sent to this module/the server a "kill signal," signaling shutdown is supposed to happen? I would say it's aptly named then. Or it should be `shutdownSignal` and `onShutdownSignal` being super explicit

Reply:
signal and onSignal it is. one moment